sym_api_client_python/clients/sym_bot_client.py

```diff
@@ -51,6 +54,16 @@ def __init__(self, auth, config):
         self.async_agent_session = None
         self.bot_user_info = None
         self.health_check_client = None
+        #Required for aiohttp to use truststore
```

Comment:
the downside here is that we are creating the ssl context even if we are not using the async calls

but at the same time, code is factored in one place and we avoid using the session object to pass the ssl context around

Reply:
Maybe we could extract that to a method `get_async_ssl_context` called only where it is needed. It would create the async SSL context the first time and reuse it next times,